app/models/course/virtual_classroom.rb

```diff
@@ -1,9 +1,21 @@
 # frozen_string_literal: true
 class Course::VirtualClassroom < ActiveRecord::Base
+  attr_writer :duration
```

Comment:
I think it better to keep end_at, this allows usage of `time_bounded_record` gem more easily